Language reworked (#2431)
WeblateOrg/weblate: docs/admin/install.rst

        # Optional deps
        pip install pytz python-bidi PyYAML pyuca

5. Create your settings (in this example it would be in
   :file:`~/weblate-env/lib/python2.7/site-packages/weblate/settings.py`
   based on the :file:`settings_example.py` in the same directory).

> 6. You can now run Weblate commands using :command:`weblate` command, see

Author: Something doesn't work using this in the text.

.. note::

    If you are running a version from Git, you should also regenerate locale
    files every time you are upgrading. You can do this by invoking the script
    :file:`./scripts/generate-locales`.

.. _install-pip:

Installing Weblate with pip
+++++++++++++++++++++++++++

If you decide to install Weblate using the pip installer, you will notice some
differences. Most importantly the command line interface is installed to the
system path as :command:`weblate` instead of :command:`./manage.py` as used in
this documentation. Also when invoking this command, you will have to specify

> settings, either by the environment variable `DJANGO_SETTINGS_MODULE` on the

Author: There is no "or" following the "either" here.

> settings, either by the environment variable `DJANGO_SETTINGS_MODULE` on the

Reviewer: ```suggestion
settings by the environment variable `DJANGO_SETTINGS_MODULE` on the
```

There used to be more ways, but right now this one is only recommended one.

.. _mysql-create:

Creating a database in MySQL
~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Create ``weblate`` user to access the ``weblate`` database:

.. code-block:: mysql

    # Grant all privileges to the user ``weblate``
    GRANT ALL PRIVILEGES ON weblate.* TO 'weblate'@'localhost'  IDENTIFIED BY 'password';

    # Create a database on MySQL >= 5.7.7
    CREATE DATABASE weblate CHARACTER SET utf8mb4;

>     # Use utf8 for older versions

Author: ```suggestion
    # Use utf-8 for older versions
```
Not sure here.

            'OPTIONS': {

>                 # In case of older MySQL server which has default MariaDB

Author: ```suggestion
                # In case of using an older MySQL server, which has MariaDB by default
```
Does it though?

Reviewer: The original comment is wrong, it should be talking about MyISAM instead of MariaDB. Fixed in 418673d62a

Correctly configure HTTPS
+++++++++++++++++++++++++

It is strongly recommended to run Weblate using the encrypted HTTPS protocol.

> After enabling it you should sed :setting:`ENABLE_HTTPS` settings, which also adjusts

Author: ```suggestion
After enabling it, you should set :setting:`ENABLE_HTTPS` in the settings, which also adjusts
```
Or the tool "sed"?

Reviewer: It should be indeed "set"
